docs/native-modules.md

```diff
@@ -1,519 +1,519 @@
----

```

Comment:
Are there any changes to this file or are there only whitespace changes? If no changes can you remove this file from this change so it doesn't cause merge conflicts with the other PRs? Thanks!